docs/index.md

```diff
@@ -247,7 +247,7 @@ collected 4 items
 ========================== 4 passed in 0.07 seconds ===========================
 ```
 
-#### *fixtures*
+#### <a id="fixtures"></a> *fixtures*
```

Comment:
I suggest rather to remove the italic here and on the previous subsection title (parameters).

Reply:
Not quite sure what you mean here?  I created the anchor link so I could reference this area further down (line 508). Even if it's not in italics, I would still want to declare an anchor? Or does mkdocs make anchors for me? (I'm used to sphinx)

Reply:
mkdocs does it automatically